content/sigs/docs/gsod/index.adoc

```diff
@@ -22,14 +22,15 @@ GSoD in Jenkins is organized by the link:/sigs/docs[Documentation special intere
 
 == GSoD 2020
 
-Jenkins project will be a mentoring organization in Google Season of Docs 2020.
-We invite technical writers to join the community and contribute to the documentation being used by millions of Jenkins users worldwide.
+The Jenkins project is a mentoring organization in Google Season of Docs 2020.
+The writing project started September 14, 2020 and will conclude November 30, 2020.
+We're looking forward to the results from the writing project and our first experience as a Google Season of Docs mentoring organization.
 
-=== Projects
+=== GSoD 2020 Project
 
 * link:/sigs/docs/gsod/2020/projects/document-jenkins-on-kubernetes[Document Jenkins on Kubernetes] by link:/blog/authors/zaycodes[Zainab Abubakar]
 
-=== Project ideas
+== Project ideas
```

Comment:
Maybe it makes sense to archive them on a new page

Reply:
Included in ee5cbf25d3cf05656577b957ce39ae4403e59bd4